string grasp_planner_name
string object_name
int64 object_id
int64 grasp_id
bool is_top_grasp
#Inputs for grasp type pgm inference
#Grasp type for inference.
#Can be: all, prec, power
string grasp_type
sensor_msgs/PointCloud2 scene_cloud
prob_grasp_planner/HandConfig init_hand_object_config
#Use constant array as initialization for BFGS 
float64[] init_config_array
string object_frame_id
#Inputs for grasp cnn inference
HandConfig init_hand_pcd_goal_config
string rgb_image_path
string depth_image_path
VisualInfo rgbd_info
---
prob_grasp_planner/HandConfig inf_hand_object_config
prob_grasp_planner/HandConfig init_sample_config
float32 inf_val
float32 init_val
bool success

